tests/test__collection_api.py

```diff
@@ -3787,6 +3787,20 @@ def test__aggregate_project_if_null(self):
         ])
         self.assertEqual([{'a': '<present_a>', 'b': '<missing_b>'}], list(actual))
 
+    def test__aggregate_project_if_null_multi_field(self):
+        self.db.collection.insert_one({'_id': 1, 'elem_a': '<present_a>'})
+        actual = self.db.collection.aggregate([
+            {'$match': {'_id': 1}},
+            {'$project': collections.OrderedDict([
+                ('_id', False),
+                ('a_and_b', {'$ifNull': ['$elem_a', '$elem_b', '<missing_both>']}),
+                ('b_and_a', {'$ifNull': ['$elem_b', '$elem_a', '<missing_both>']}),
+                ('b_and_c', {'$ifNull': ['$elem_b', '$elem_c', '<missing_both>']}),
+            ])}
+        ])
+        self.assertEqual([{'a_and_b': '<present_a>', 'b_and_a': '<present_a>',
```

Comment:
Please break after `self.assertEqual(`

Reply:
Note that I added a helper `get_mongodb_version` that checks the version of MongoDB
I added the newer $ifNull syntax that is supported only on versions higher than v4.4 which is currently used in this project. 
When this project upgrades into MongoDB v5, the test will pass and the syntax can be added

Do you think the PR can be approved if the aggregation fails depending on the MongoDB version